plugins/deploy_env_vars/app/views/samson_deploy_env_vars/_deploy_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<fieldset>
+  <legend>
+    Specific Environment Variables for this deploy <%= additional_info SamsonEnv::HELP_TEXT %>
+  </legend>
+
+  <% current_vars = form.object.environment_variables %>
+  <%= form.fields_for :environment_variables, current_vars << EnvironmentVariable.new do |fields| %>
+    <div class="form-group">
```

Comment:
the whole section smells like copy-paste ... can we reuse a partial instead ?

Reply:
yeah! I actually removed the warning from the `flay` rake task :blush: I will move this to a partial